src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -117,6 +130,14 @@ export default class AddStaffNotationDialog extends Mixins(DateMixin) {
   /** Flag to enable validation in this component */
   private enableValidation = false
 
+  private get administrativeDissolution (): boolean {
```

Comment:
Since this component already imports mixins, you can import the Enum Mixin and use the new filing type helpers you added, instead of duplicating them here 😃 

(The helpers expect an object with property 'name', so you could call it like this: `isTypeAdministrativeDissolution({ name: this.name })`.)